invenio_app_ils/cli.py

```diff
@@ -92,6 +92,18 @@ def get_documents_items(internal_locations, n_docs, n_items):
             "medium": "{}".format(ITEM_MEDIUMS[randint(0, LEN_ITEM_MEDIUMS)]),
             "status": "{}".format(ITEM_STATUSES[randint(0, LEN_ITEM_STATUSES)]),
         }
+        # import ipdb; ipdb.set_trace()
+
+        item["document"] = {
+            "$ref": "{scheme}://{host}/api/resolver/items/document/{pid_value}"
+            .format(
+                scheme="https",
+                host="127.0.0.1:5000",
```

Comment:
I guess at some point we will decide to replace this with some other fake domain name and we need to align it with the config `JSONSCHEMA_HOST`.

Reply:
I will point to the config then!